### PR TITLE
fix: user externalPg.url instead of pg

### PR DESCRIPTION
**Pull request by zll600**

### CODE REVIEW

File: content/docs/get-started/self-host.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ For production setup, you should configure a proper [External URL](/docs/get-sta
 helm -n <YOUR_NAMESPACE> \
 --set "bytebase.option.port"={PORT} \
 --set "bytebase.option.external-url"={EXTERNAL_URL} \
---set "bytebase.option.pg"={PGDSN} \
+--set "bytebase.option.externalPg.url"={PGDSN} \
```
Comment (tianzhou) [inline content/docs/get-started/self-host.md:242]: Using a shorter flag is intentional as it's more memorable and allows fewer mistakes.

The new variable is long and causes more ambiguities, to name a few similar names.

externalPg.url,  external-pg.url,  external-pg-url.

Also our main bytebase app uses --pg, thus it's less consistent using a different name here.

Comment (tianzhou) [reply content/docs/get-started/self-host.md:242]: The `pg` flag is less obvious than the `port` flag. This is the nature of most flags. Regardless of the flag name, developers need to check further to see the exact format.

Comment (zll600) [reply content/docs/get-started/self-host.md:242]: `externalPg.url` comes from our [helm-chart](https://github.com/bytebase/bytebase/blob/3d7a21dbabb1b37ef860f285baf5ddada0f15ee3/helm-charts/bytebase/values.yaml#L14C14-L14C14). need @h3n4l to help review.

Comment (tianzhou) [reply content/docs/get-started/self-host.md:242]: Oh, never mind, then that's a different story.

Comment (tianzhou) [reply content/docs/get-started/self-host.md:242]: Then please provide the PR description next time to provide more contetx

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**